packages/x-components/src/x-modules/queries-preview/components/query-preview.vue

```diff
@@ -154,6 +137,10 @@
     @Inject({ default: undefined })
     protected location?: FeatureLocation;
 
+    protected get query(): string {
+      return this.queryPreviewInfo.query;
+    }
+
```

Comment:
Add doc, something like this
`/**
     * The query to make the queryPreviewRequest's with
     *
     * @returns The search query string.
     * @internal
     */`